devicetypes/smartthings/zwave-siren.src/zwave-siren.groovy

```diff
@@ -64,12 +64,17 @@ metadata {
 def installed() {
 	// Device-Watch simply pings if no device events received for 122min(checkInterval)
 	sendEvent(name: "checkInterval", value: 2 * 60 * 60 + 2 * 60, isStateChanged: true, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID])
+	runIn(10, "initialize", [overwrite: true, forceForLocallyExecuting: true])
```

Comment:
Why not 0 delay? Unless I do not the devices behavior well, I think sending the commands to the hub is much better idea specially because after a device is added, the hub reserves a period of time where the messages for recently added node will have higher priority and will then be sent ahead of other messages. 

My idea would be to start a timer and check every 10 seconds for about a few minutes to see if we have received the data that we wanted or else issue new commands. I think this will prove reliability and certainty that we have what we wanted.

Reply:
@kianooshST That is an interesting idea. I can do this much like what I did for the lock. Actually, I am beginning to wonder if that might be better behavior to do in general. We kind of do it with WakeUpNotification for Z-Wave devices, but for devices that don't knock on the door we end up making a lot of assumptions.